packages/debugger/lib/solidity/selectors/index.js

```diff
@@ -330,11 +328,25 @@ let solidity = createSelectorTree({
      * HACK: this assumes we're not about to change context! don't use this if
      * we are!
      * ALSO, this may return undefined, so be prepared for that
+     * NOTE: this selector is now somewhat misnamed, as mapped instructions are
+     * now also skipped over if they point to an internal source
```

Comment:
Could we make a new, better named selector for this, and switch the code to use that? We can leave this one here as a pass-through for backwards compatibility.

Reply:
I could also just rename the selector, it's pretty internal.  For some reason I didn't think of that while writing this initially.  But yes if we really want compatibility we could do that.

Reply:
OK, renamed this to `nextUserStep` instead.